config/initializers/raven.rb

```diff
@@ -0,0 +1,9 @@
+# Configure Sentry-Raven if raven_dsn secret is set
+# frozen_string_literal: true
```

Comment:
I think the linter will only accept this as the very first line of this file.

Reply:
Should be fixed